app/src/parser.c

```diff
@@ -92,73 +92,83 @@ parser_error_t parser_getNumItems(const parser_context_t *ctx, uint8_t *num_item
 
 __Z_INLINE bool_t parser_areEqual(uint16_t tokenIdx, const char *expected) {
     if (parser_tx_obj.tx_json.json.tokens[tokenIdx].type != JSMN_STRING) {
-        return bool_false;
+        return false;
     }
 
     int32_t len = parser_tx_obj.tx_json.json.tokens[tokenIdx].end - parser_tx_obj.tx_json.json.tokens[tokenIdx].start;
     if (len < 0) {
-        return bool_false;
+        return false;
     }
 
     if (strlen(expected) != (size_t) len) {
-        return bool_false;
+        return false;
     }
 
     const char *p = parser_tx_obj.tx_json.tx + parser_tx_obj.tx_json.json.tokens[tokenIdx].start;
     for (int32_t i = 0; i < len; i++) {
         if (expected[i] != *(p + i)) {
-            return bool_false;
+            return false;
         }
     }
 
-    return bool_true;
+    return true;
 }
 
 __Z_INLINE bool_t parser_isAmount(char *key) {
     if (strcmp(key, "fee/amount") == 0) {
-        return bool_true;
+        return true;
```

Comment:
function return type must be bool not bool_t

Reply:
Fixed that one and other two I had forgotten !